src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -2066,11 +2066,10 @@ HRESULT CEndlessUsbToolDlg::OnLiveUsbClicked(IHTMLElement* pElement)
 {
     FUNCTION_ENTER;
 
-	SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
+    SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
```

Comment:
There's a few more occurrences in the file. As said: if it's not annoying to do, it would be nice to get the diff to have only the actual changes (super low priority!)